spec/buftok_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'filewatch/buftok'
+
+describe FileWatch::BufferedTokenizer do
+
+  context "when using the default delimiter" do
+    subject { FileWatch::BufferedTokenizer.new }
```

Comment:
This is not bad, but just a curiosity. If you use this syntax and also define the describe the way you did in line 3, you're basically doing the same.